soundbay/train.py

```diff
@@ -44,6 +44,7 @@ def modeling(
     scheduler_args,
     model_args,
     logger,
+    num_unfrozen,
```

Comment:
1. Should have a default = None, in which case everything is unfrozen. 
2. Add this parameter to docstring (after logger)

Reply:
I've changed the way this method and parameter behave according to Moshe's request (it now has True / False / None values)

about 2 - is printing a statement about being in finetune mode a "good enough" log? (shows up in "logs" in wandb)
![image](https://user-images.githubusercontent.com/62894318/164990446-200526ac-0701-401c-9bc7-4c5f863edde3.png)